app/west.yml

```diff
@@ -4,12 +4,14 @@ manifest:
       url-base: https://github.com/zephyrproject-rtos
     - name: zmkfirmware
       url-base: https://github.com/zmkfirmware
+    - name: megamind4089
+      url-base: https://github.com/megamind4089
     - name: microsoft
       url-base: https://github.com/microsoft
   projects:
     - name: zephyr
-      remote: zmkfirmware
-      revision: v2.5.0+zmk-fixes
+      remote: megamind4089
+      revision: v2.5.0+zmk-fixes-stm32f4
```

Comment:
Anything here actually relevant to this work that this is needed?

Reply:
No, please ignore this file. Will remove this before final cleanup.
Just added here to show that i had to cherry-pick this PR to make it work in half duplex mode.
https://github.com/zephyrproject-rtos/zephyr/pull/40525